src/main/java/com/github/onsdigital/logging/layouts/style/Beautifier.java

```diff
@@ -85,32 +77,30 @@ private static ColourConfiguration getColourMapping(Level level) {
         }
     }
 
-    public static String beautifyParameters(ILoggingEvent event, LogParameters parameters) {
-        if (parameters == null || parameters.getParameters() == null || parameters.getParameters().isEmpty()) {
+    public static String beautifyParameters(ILoggingEvent e, LogParameters p) {
+        if (p == null || p.getParameters() == null || p.getParameters().isEmpty()) {
             return "";
         }
 
-        ColourConfiguration configuration = getColourMapping(event.getLevel());
-        StringBuilder result = new StringBuilder(applyColour(configuration.getPrimaryColour(), "parameters={ "));
+        Colourizer c = getColourizer(e.getLevel());
+        StringBuilder result = new StringBuilder(c.primaryColour(PARAMS_KEY));
 
-        Iterator<Map.Entry<String, Object>> iterator = parameters.getParameters().entrySet().iterator();
+        Iterator<Map.Entry<String, Object>> iterator = p.getParameters().entrySet().iterator();
         Map.Entry<String, Object> entry = null;
         while (iterator.hasNext()) {
             entry = iterator.next();
-            result.append(START_TAG)
-                    .append(applyColour(configuration.getPrimaryColour(), entry.getKey() + "="))
-                    .append(applyColour(configuration.getSecondaryColour(), entry.getValue().toString()))
-                    .append(iterator.hasNext() ? applyColour(configuration.getSecondaryColour(), ", ") : "");
+            if (entry.getValue() != null && !StringUtils.isEmpty(entry.getValue().toString())) {
+                result.append(c.secondaryColour(entry.getKey()))
+                        .append(c.tertiaryColour("="))
+                        .append(c.secondaryColour(entry.getValue().toString()))
+                        .append(iterator.hasNext() ? c.primaryColour(", ") : "");
+            }
         }
-        result.append(applyColour(configuration.getPrimaryColour(), " }"));
+        result.append(c.primaryColour("}"));
```

Comment:
seems odd to be appending `}` here without appending `{` also in this function. I am guessing its because its the end of a json log? Consider moving it out of here

Reply:
Ah I see that the start `{` is added by `PARAMS_KEY`. Consider also declaring the end curly brace as a const, potentially also separating the opening curly brace to be explicit about where it is.